ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-master__upgrade.yaml

```diff
@@ -0,0 +1,37 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-operator-upgrade
```

Comment:
How about cover fips env?

Reply:
Makes sense to me. We can do this by adding `FIPS_ENABLED: "true"` to the workflow's env.